src/Components/Search/Previews/Grids/ArtistSearch/__tests__/MarketingCollections.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { Serif } from "@artsy/palette"
+import { mount } from "enzyme"
+import React from "react"
+
+import { CollectionTitles } from "../MarketingCollections"
+
+describe("CollectionTitles", () => {
+  it("with no colons", () => {
+    const wrapper = mount(<CollectionTitles title="The best one" />)
+    expect(wrapper.find(Serif).length).toEqual(1)
```

Comment:
Maybe add an additional `expect(...)` that checks against the passed title ("with no colons")? Otherwise it might be a little clearer if the test described testing page structure since that seems to be the intent:

```js
it('renders one Serif element if no colon in text', () => { ... })
```